src/Main.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Main extends React.Component {
+    render() {
+        return;
```

Comment:
Did you end up using the Main component for anything? Or is this pending for further design improvements?

Reply:
Yes, this was just boilerplate. Once the children components were made, I put them in Main.